doc/Makefile

```diff
@@ -0,0 +1,28 @@
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+clean:
+	@echo "Removing everything under 'api', 'doc', and '_build'..."
+	-@rm -rf api doc _build
```

Comment:
The Sphinx default implementation only cleaned the contents of `_build`. The `apidoc` extension generates the `api` subdirectory and the workaround target below generates the `doc` subdirectory, both of which should also be cleaned up.